Gems/Atom/RHI/Vulkan/Code/Source/RHI/Instance.cpp

```diff
@@ -79,48 +79,57 @@ namespace AZ
 #if defined(AZ_VULKAN_USE_DEBUG_LABELS)
             m_descriptor.m_optionalExtensions.push_back(VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
 #endif
-            
-            uint32_t apiVersion = VK_API_VERSION_1_0;
-            // vkEnumerateInstanceVersion is a Vulkan 1.1 function 
+
+            uint32_t appApiVersion = VK_API_VERSION_1_0;
+            m_instanceVersion = VK_API_VERSION_1_0;
+            // vkEnumerateInstanceVersion is a Vulkan 1.1 function
             // so if it's not available we assume Vulkan 1.0
-            if (GetContext().EnumerateInstanceVersion && GetContext().EnumerateInstanceVersion(&apiVersion) != VK_SUCCESS)
+            if (GetContext().EnumerateInstanceVersion)
             {
-                AZ_Warning("Vulkan", false, "Failed to get instance version.");
-                return false;
+                if (GetContext().EnumerateInstanceVersion(&m_instanceVersion) != VK_SUCCESS)
+                {
+                    AZ_Warning("Vulkan", false, "Failed to get instance version.");
```

Comment:
Should this be an error? There looks like a lot of initialization that is skipped if this fails.